module-2/backend/week_three.md

```diff
@@ -10,14 +10,63 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+rails new (And maybe the option to not use Spring. Spring breaks dreams.)
+
+
 2. What do Models generally inherit from in rails?
+
+ActiveRecord::Base
+
+
 3. What do Controllers generally inherit from in a rails project?
+
+ApplicationController
+
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+resources :horses, only: [:show]
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+
+rake routes gives you information about the prefix, verb, uri pattern, and corresponding controller action.
+
+
 6. What is an example of a route helper? When would you use them?
+
+A route helper is something like horses_path, which would bring you to the index of horses. You use them in place of writing out a url.
+
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+path is relative. url is absolute.
+
+
 8. What are strong params and why are the necessary?
+
+Strong params prevent parameters from being mass assigned to an ActiveRecord object as a security measure. You must specify which params are allowed to be assigned.
+
+
 9. What role does `form_for` play in helping us create our forms?
+
+form_for makes it easy to create forms to create and update ActiveRecord objects.
+
 10. How does `form_for` know where to submit the user's input?
-11. Create a form using a `form_for` helper to create a new `Horse`. 
+
+An instance of a model is passed in when form_for is called. It creates or updates that instance.
```

Comment:
Right general idea, but it's important to think about how this whole cycle works. `form_for` will take the parameter that's passed to it and generate an HTML form that will submit to the appropriate route using the appropriate method (HTTP verb). Your URI pattern in your `rake routes` output will tell you what that route/method is. View source on one of the forms you create to see this in action.